packages/teleport/src/services/audit/types.ts

```diff
@@ -49,6 +49,9 @@ export const eventCodes = {
   APP_SESSION_CHUNK: 'T2008I',
   APP_SESSION_START: 'T2007I',
   APP_SESSION_END: 'T2011I',
+  APPLICATION_CREATED: 'TAP03I',
+  APPLICATION_UPDATED: 'TAP04I',
+  APPLICATION_DELETED: 'TAP05I',
```

Comment:
This is definitely a nitpick, but can we name these `APP_CREATED`, `APP_UPDATED`, etc. so they're consistent with event codes that are already defined here?

Reply:
Sure, no prob!